api/lib/middleware/auth.ts

```diff
@@ -1,28 +1,21 @@
-import axios from 'axios'
-const jwtDecode = require('jwt-decode')
-import { config } from '../../../config'
+import * as jwtDecode from 'jwt-decode';
+import { config } from '../../../config';
 
-module.exports = (req, res, next) => {
-    const userId = req.headers[config.cookies.userId] || req.cookies[config.cookies.userId]
-    const jwt = req.headers.authorization || req.cookies[config.cookies.token]
-
-    console.log(jwt);
-    const jwtData = jwtDecode(jwt)
-    const expires = new Date(jwtData.exp).getTime()
-    const now = new Date().getTime() / 1000
-    const expired = expires < now
+export function authInterceptor(req, res, next) {
```

Comment:
and also exports rather than module.export =

Reply:
is there a reason for this? just curious as pdf.js uses modue.export

Reply:
it's the JavaScript way. Node.js doesn't natively support ecmascript modules but TypeScript transpiles it. My guess is that this project was written in JS and was copied over to TS.